docs/user-guide/cli-install-configure-zosmf-security.md

```diff
@@ -0,0 +1,59 @@
+
+:::
+
+The following table details the required z/OSMF REST services and examples of the Zowe CLI **[correct?]** features they enable. If the profile validation command returns any errors, use this table to find IBM documentation for the z/OSMF REST APIs.
+
+| z/OSMF REST Service        | REST Endpoint | Description | More information |
+| ----------- | ----------- | ---------- | ------------- |
+| Cloud provisioning services | Endpoints that begin with: `/zosmf/provisioning/` | Cloud provisioning for development environments. | <ul><li>[Cloud provisioning services](https://www.ibm.com/docs/en/zos/2.5.0?topic=services-cloud-provisioning)</li></ul> |
+| TSO/E address space services | Endpoints that begin with: `/zosmf/tsoApp` | TSO commands (`zowe zos-tso issue`). | <ul><li>[TSO/E address space services](https://www.ibm.com/docs/en/zos/2.5.0?topic=services-tsoe-address-space)</li><li>[Class activations that z/OSMF requires](https://www.ibm.com/docs/en/zos/2.2.0?topic=guide-security-configuration-requirements-zosmf#DefaultSecuritySetupForZosmf__ResourceAuthorizationsForRESTapi)</li></ul> |
+| z/OS console services | Endpoints that begin with: `/zosmf/restconsoles/`<br/> Example: `/zosmf/restconsoles/defcn` | Console commands (`zowe zos-console issue`). Any MVS console command such as MODIFY and DISPLAY. | <ul><li>[z/OS console services](https://www.ibm.com/docs/en/zos/2.5.0?topic=services-zos-console)</li><li>[Updating your system for the z/OS console REST interface](https://www.ibm.com/docs/en/zos/2.2.0?topic=time-updating-your-system-zos-console-rest-interface)</li><li>[Resource authorizations for the z/OS console services REST interface](https://www.ibm.com/docs/en/zos/2.2.0?topic=guide-security-configuration-requirements-zosmf#DefaultSecuritySetupForZosmf__zOSConsolesRestAPI)</li></ul> |
+| z/OS data set and file REST interface | Endpoints that begin with: `/zosmf/restfiles/` Example: `/zosmf/restfiles/ds/<dsname>` | Create data sets (`zowe zos-files create`), delete data sets (`zowe zos-files delete`), read (download) data sets (`zowe zos-files download`), and write (upload) data sets (`zowe zos-files upload`). <br/>Access to access method services (IDCAMS) (`zowe zos-files invoke access-method-services`). | <ul><li>[z/OS data set and file REST interface](https://www.ibm.com/docs/en/zos/2.5.0?topic=services-zos-data-set-file-rest-interface)</li><li>[Updating your system for the z/OS data set and file REST interface](https://www.ibm.com/docs/en/zos/2.2.0?topic=czft-updating-your-system-zos-data-set-file-rest-interface)</li><li>[Resource authorizations for the z/OS data set and file REST interface](https://www.ibm.com/docs/en/zos/2.2.0?topic=guide-security-configuration-requirements-zosmf#DefaultSecuritySetupForZosmf__ResourceAuthorizationsForRESTdsfilesAPI)</li></ul> |
```

Comment:
@adam-wolfe @zFernand0 -- I couldn't five the v2.5 equivalent for the v2.2 article "Updating your system for the z/OS data set and file REST interface" in the IBM Docs. Could one of you help me find a replacement?
Same for the "Resource authorizations for the z/OS data set and file REST interface" article

Reply:
https://www.ibm.com/docs/en/zos/2.5.0?topic=guide-security-structures-zosmf#DefaultSecuritySetupForZosmf__ResourceAuthorizationsForRESTdsfilesAPI__title__1

Reply:
I believe they removed almost all installation and configuration documentation. The only documentation that I can find that is relatively close to this is the documentation for IZUPRM, which is basically the z/OSMF configuration file.
https://www.ibm.com/docs/en/zos/2.5.0?topic=system-izuprmxx-reference-information

Reply:
I think it's best to simply remove this bullet item then, thank you